xbmc/GUIInfoManager.cpp

```diff
@@ -2540,6 +2540,8 @@ const infomap musicplayer[] =    {{ "title",            MUSICPLAYER_TITLE },
                                   { "samplerate",       MUSICPLAYER_SAMPLERATE },
                                   { "codec",            MUSICPLAYER_CODEC },
                                   { "discnumber",       MUSICPLAYER_DISC_NUMBER },
+                                  { "discname",         MUSICPLAYER_DISC_NAME },
```

Comment:
"disctitle" seems better than "discname" - people have names, songs  and discs have titles, and the data comes from DISCSUBTITLE tag. Going to live with the info label a long time so better to be happy with naming from begining.

Reply:
Also need to add comment to document addition
e.g. @skining-v19 ** [New Infolabel]**